cmd/crc/cmd/console.go

```diff
@@ -51,15 +58,16 @@ func runConsole(arguments []string) {
 		output.Outf("To login as an admin, run 'oc login -u kubeadmin -p %s %s'\n", result.ClusterConfig.KubeAdminPass, result.ClusterConfig.ClusterAPI)
 	}
 	if consolePrintURL || consolePrintCredentials {
-		return
+		return nil
 	}
 
 	if !machine.IsRunning(result.State) {
-		errors.ExitWithMessage(1, "The OpenShift cluster is not running, cannot open the OpenShift Web Console.")
+		return errors.New("The OpenShift cluster is not running, cannot open the OpenShift Web Console")
```

Comment:
@guillaumerose since you removed this `dot` one of the integration test is failing, can you update the integration test for same?

Reply:
Thanks, I changed the test.